src/apps/recipe/graphql/queries.js

```diff
@@ -176,3 +132,31 @@ export const RECIPE = gql`
       }
    }
 `
+
+export const SACHET_ITEMS = gql`
+   {
```

Comment:
```suggestion
   query {
```

Make sure to pass the keyword, by default it is taken to be a query but passing it would help other's who doesn't know about it